web/client/localConfig.json

```diff
@@ -345,7 +345,10 @@
                 }
             },
             "AddGroup",
-            "TOCItemsSettings",
+            {
+                "name": "TOCItemsSettings",
+                "parentPlugin": "TOC"
```

Comment:
I don't like this: we already have containement information through the plugin containers configuration. Can you try to use that instead?

Reply:
I think @vlt1 need it to create this kind of structure in the mockup, isn't it?
![image](https://user-images.githubusercontent.com/1279510/70700573-446aa500-1ccb-11ea-9218-985dddaab805.png)

In this case LayerSetting(TOCItemsSettings), as well as AttributeTable (FeatureEditor) is not related by plugins settings to the TOC. Anyway I suggest to create a separate configuration that defines the structure of the UI, strictly related to the ContextEditor, that defines parent-child relationship. 
Also the different labels (Layer Settings for TOCItemsSettings, Attribute Table for FeatureEditor and so on) should be stored somewhere.